.cicd/run.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+. $HELPERS_DIR/execute.sh
+
+execute mkdir -p $BUILD_DIR
+
+if [[ $(uname) == Darwin ]]; then
+
+    MAC_TEST="ctest -j$JOBS --output-on-failure -T Test"
+
+    cd $BUILD_DIR
+    execute ccache -s
+    if [[ $ENABLE_BUILD == true ]] || [[ $TRAVIS == true ]]; then
+        [[ $TRAVIS == true ]] && execute mkdir -p wasms # Travis doesn't have git access to clone the private repo (Temporary)
+        execute cmake -DCMAKE_BUILD_TYPE=Release ..
+        execute make -j$JOBS
```

Comment:
We moved our builders off the Kubernetes "docker on docker" fleet, so let's get rid of the `$JOBS` variable.

Reply:
We use `export JOBS=${JOBS:-"$(getconf _NPROCESSORS_ONLN)"}` in general.sh